plugins/create/lang/en.js

```diff
@@ -6,7 +6,7 @@
  * Author: 
  */
 
- theUILang.mnu_create			= "Create Torrent...";
+ theUILang.mnu_create			= "Create Torrent";
```

Comment:
Could you submit a separate pull request for this change?

Reply:
A separated pull request has been submitted [here](https://github.com/Novik/ruTorrent/pull/2688)